core/src/jsMain/kotlin/FilterSet.kt

```diff
@@ -0,0 +1,45 @@
+package com.juul.kable
+
+import com.benasher44.uuid.Uuid
+import com.juul.kable.external.BluetoothLEScanFilterInit
+import com.juul.kable.external.BluetoothManufacturerDataFilterInit
+
+public data class FilterSet(
+    public val services: List<Filter.Service> = emptyList(),
+    public val name: Filter.Name? = null,
+    public val namePrefix: Filter.NamePrefix? = null,
+    public val manufacturerData: List<Filter.ManufacturerData> = emptyList(),
```

Comment:
To be complete, we would have a `List<Filter.ServiceData>` property here as well, but seeing as that is not implemented in the browser I have omitted it to prevent people from attempting to use it (and subsequently finding out it doesn't work).